psp/src/vram_alloc.rs

```diff
@@ -27,14 +29,20 @@ impl VramAllocatorSingleton {
     }
 }
 
-pub struct VramMemChunk {
+pub struct VramMemChunk<'a> {
     start: u32,
     len: u32,
+    // Needed since VramMemChunk has a lifetime, but doesn't contain references
+    vram: PhantomData<&'a mut ()>,
```

Comment:
Not an issue, but `PhantomData<&'a ()>` suits better imo.

Reply:
Yeah either would work.